lib/appmap/builtin_hooks/yaml.yml

```diff
@@ -8,13 +8,12 @@
   - Psych#parse_stream
   label:
     - format.yaml.parse
-    - deserialize
     - deserialize.unsafe
```

Comment:
Actually `parse` methods are safe -- they don't build objects but simply return the document tree or stream. In fact they can be removed from here -- they're not deserialization but parsing. `format.yaml.parse` seems like an adequate label (but only for these: `load` functions actually use `parse`), though I'm not sure about its utility.

Whole psych should probably be labeled something like:
```yaml
format.yaml.parse: Psych.parse_stream  # parse actually uses this one
deserialize.unsafe: Psych.load, Psych.load_stream  # or Psych::Nodes::Node#to_ruby, though this seems like overkill
deserialize.safe: Psych.safe_load
```

Not sure about

Reply:
OK, I'll update it.